source/Plugins/Language/Swift/SwiftLanguage.cpp

```diff
@@ -563,6 +563,8 @@ static void LoadSwiftFormatters(lldb::TypeCategoryImplSP swift_category_sp) {
                    ConstString("CoreGraphics.CGFloat"), summary_flags);
   AddStringSummary(swift_category_sp, "${var.native}",
                    ConstString("Foundation.CGFloat"), summary_flags);
+  AddStringSummary(swift_category_sp, "${var.native}",
```

Comment:
So, is `CGFloat` living exclusively in `FoundationBase`? If so, the other formatter can be removed.

Reply:
Only in swift-corelibs-foundation. Does this cover Darwin too?

Reply:
Also, I'd like to be able to debug things built with swift-corelibs-foundation and Swift 4.x :)

Reply:
This covers everything (Darwin and Linux). If you really want to match both, use a regex instead of duplicating the code (see other examples, or ping me if you have issues).

Also, compiler and debugger are revlocked until swift grows a stable module format so the backcompatibility argument doesn't hold.

Thanks!